src/style.css

```diff
@@ -0,0 +1,122 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.card {
```

Comment:
11. [FILE STRUCTURE] - You should create a separate file for each BEM block

Reply:
Agree, fix @Andrew-Skoryk comment